manifests/autotune-configs/hotspot-micrometer-config.yaml

```diff
@@ -5,22 +5,20 @@ metadata:
 layer_name: hotspot
 layer_level: 1
 details: hotspot tunables
-layerPresence:
-  query:
-    datasource:
-    - name: prometheus
-      query: 'jvm_memory_used_bytes{area="heap",}'
-      key: pod
+layer_presence:
+  queries:
+  - datasource: prometheus
```

Comment:
Prometheus does not have single quotes here. Do we need id="Tenured Gen", I 'm updating the tests in the tests it is present. Should I remove it?